File: 002-Backend/LaPlata.API/LaPlata.API/LaPlata.Domain/DTOs/Despesas/Despesa/ReadDespesaDTO.cs
namespace LaPlata.Domain.DTOs
{
    public class ReadDespesaDTO
    {
        public int Id { get; set; }
        public string Descricao { get; set; }
        public int ValorInteiro { get; set; }
        public int? ValorCentavos { get; set; }
        public DateTime DataDespesa { get; set; }
        public int? QtdParcelas { get; set; }
        public ReadCategoriaDTO Categoria { get; set; }
    }
}

File: 003-Frontend/HomeTools.Front/src/app/dto/financas/despesas/readDespesaDto.ts
import { Categoria } from "src/app/interfaces/categoria";

export interface ReadDespesaDto {
  Id: number;
  Descricao: string;
  ValorInteiro: number;
  ValorCentavos: number;
  DataDespesa: Date;
  QtdParcelas: number;
  Categoria: Categoria;
}
